### PR TITLE
add target to build macOS universal binary

### PR DESCRIPTION
**Pull request by anjannath**

### CODE REVIEW

**Comment by cfergeau** [comment]
I came up with https://github.com/cfergeau/vfkit/blob/universal/Makefile for the Makefile, both similar to and different from yours :)

**Comment by anjannath** [comment]
pushed your commit to this branch :)

**Comment by cfergeau** [comment]
Well, ack from me, if it's fine with you as well we can push it!

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
-.PHONY: build
-build:
-	go build -o vfkit ./cmd/vfkit
+# the go compiler is doing a good job at not rebuilding unchanged files
+# this phony target ensures out/vfkit-* are always considered out of date
+# and rebuilt. If the code was unchanged, go won't rebuild anything so that's
+# fast. Forcing the rebuild ensure we rebuild when needed, ie when the source code
+# changed, without adding explicit dependencies to the go files/go.mod
+.PHONY: force-build
+force-build:
```
Comment (praveenkumar) [inline Makefile:26]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**